doc/ref_arch/openstack/chapters/chapter08.rst

```diff
@@ -178,8 +178,6 @@ Here are a couple of publicly available playbooks :
    verification <https://git.opnfv.org/functest/plain/ansible/site.yml?h=stable/wallaby>`__
 -  `Anuket
    RC1 <https://git.opnfv.org/functest/plain/ansible/site.cntt.yml?h=stable/wallaby>`__
--  `Kubernetes
```

Comment:
why are we deleting -- OpenStack deployment now requires Kubernetes

Reply:
@pgoyal01 @collivier In my understanding, even if the OpenStack services are containerised and deployed with Kubernetes, only RC1 test suites will be applied for conformance. Can you confirm?

Reply:
For this scenario, RC2 or Functest Kubernetes could be executed vs Kubernetes before OpenStack deployment.
But as it's an operator choice ; only RC1 can be mandatory in RA1 doc. 

There is nothing wrong if Kubernetes is listed in Functest. It's a kind of introduction of all Functest capabilities. Not only the ones used by RC1.

I'm fine if you want to remove all refs to K8s to.

Reply:
@collivier @pgoyal01 Please review the modification